rest/route/service_test.go

```diff
@@ -659,9 +659,11 @@ func TestTestPaginator(t *testing.T) {
 						TaskId:    model.ToStringPtr(""),
 						TestFile:  model.ToStringPtr(""),
 						Logs: model.TestLogs{
-							URL:    model.ToStringPtr(""),
-							URLRaw: model.ToStringPtr(""),
-							LogId:  model.ToStringPtr(""),
+							URL:            model.ToStringPtr(""),
+							URLRaw:         model.ToStringPtr(""),
+							LogId:          model.ToStringPtr(""),
+							RawDisplayURL:  model.ToStringPtr(""),
```

Comment:
This is fine, but in the future you can leave these fields out (assuming you don't care that it will be a null value)